netcontrol/Dockerfile.dev

```diff
@@ -0,0 +1,15 @@
+FROM python:3.12-alpine3.19
```

Comment:
If the Dockerfile.dev is the same as the dockerfile, why not using a single file ?

Reply:
I just did this in case we needed a different setup for them later